web/packages/teleport/src/types.ts

```diff
@@ -63,7 +63,7 @@ export enum NavTitle {
   // Access Management
   Users = 'Users',
   Bots = 'Bots',
-  Roles = 'User Roles',
+  Roles = 'Roles',
```

Comment:
I agree that this makes more sense, but I know we're going to get support tickets on this one 😂

Reply:
yeah, this was a specific request from Kenny and it's how it is in the figma, hopefully no complaints 🤞